packages/toolpad-app/src/theme.ts

```diff
@@ -1,104 +1,723 @@
-import { createTheme } from '@mui/material/styles';
-import { red } from '@mui/material/colors';
-import type {} from '@mui/x-data-grid-pro/themeAugmentation';
-import type {} from '@mui/lab/themeAugmentation';
+import { deepmerge } from '@mui/utils';
+import ArrowDropDownRounded from '@mui/icons-material/ArrowDropDownRounded';
+import { createTheme, ThemeOptions, Theme, alpha } from '@mui/material/styles';
 
```

Comment:
One bug, we miss a `color-schema`

<img width="1197" alt="Screenshot 2022-09-27 at 15 54 06" src="https://user-images.githubusercontent.com/3165635/192545924-4dd0954c-309a-4e89-98ec-662892635e3e.png">

https://toolpad-pr-988.onrender.com/_toolpad/app/cl44kcwjj34669yofihq14yr0/pages/w713g5e

Reply:
I've added a `color-scheme` CSS variable to our `DataGrid`